cli/tests/unit/serve_test.ts

```diff
@@ -867,10 +926,10 @@ Deno.test(
     let responseText = new TextDecoder("iso-8859-1").decode(buf);
     clientConn.close();
 
-    assert(/\r\n[Xx]-[Hh]eader-[Tt]est: Æ\r\n/.test(responseText));
-
     ac.abort();
     await server;
+
+    assertMatch(responseText, /\r\n[Xx]-[Hh]eader-[Tt]est: Æ\r\n/);
```

Comment:
Why was it moved?

Reply:
If this assertion fails, it causes the test to fail with a resource leak instead of the actual error which made it kind of annoying.